crates/merod/src/cli/init.rs

```diff
@@ -291,6 +294,42 @@ impl InitCommand {
                 .insert("icp".to_owned(), local_config);
         }
 
+        {
+            let _ignored = client_params.insert(
+                "stellar".to_owned(),
+                ClientConfigParams {
+                    network: "Test SDF Network ; September 2015".into(),
+                    protocol: "stellar".into(),
+                    contract_id: "CDOQVQE4UARLUXZHOT5YVKTE4UV4CT5PK6UXYOHILWQNSSO4POGJF6PS"
+                        .parse()?,
+                    signer: ClientSelectedSigner::Local,
+                },
+            );
+
+            let mut local_config = ClientLocalConfig {
+                signers: Default::default(),
+            };
+
+            let _ignored = local_config.signers.insert(
+                "Public Global Stellar Network ; September 2015".to_owned(),
+                generate_local_signer(
+                    "https://soroban.stellar.org".parse()?,
+                    ConfigProtocol::Stellar,
+                )?,
+            );
+
+            let _ignored = local_config.signers.insert(
+                "Test SDF Network ; September 2015".to_owned(),
+                generate_local_signer(
+                    "https://soroban-testnet.stellar.org".parse()?,
+                    ConfigProtocol::Stellar,
+                )?,
+            );
```

Comment:
let's try different network names

Reply:
testnet / mainnet and then later i transform it with Network::testnet() Network::public() from libs